zone/zone.cpp

```diff
@@ -605,26 +605,13 @@ void Zone::LoadTempMerchantData()
 
 void Zone::LoadNewMerchantData(uint32 merchantid) {
 
-	std::list<MerchantList> merlist;
+	std::list<MerchantList> merchant_list;
 
-	std::string query = fmt::format(
-		SQL(
```

Comment:
Any reason we scrunched up the query (readability) ? Repository usage would also clean this up

Reply:
I can put it back the way it was, I figured it was more readable this way.